docs/howto.md

```diff
@@ -100,7 +100,7 @@ Here our query for database. There is a text with `_SQL` literal and single para
 Here is `request()` asynchronous function call.
 
 ```cpp
-ozo::request(ozo::make_provider(io, conn_info), query, ozo::into(res),
+ozo::request(ozo::make_connector(io, conn_info), query, ozo::into(res),
```

Comment:
There is different arguments order for `make_connector`.

Reply:
**Done**